Project.toml

```
@@ -48,7 +48,7 @@ MeshIntegrals = "0.16, 0.17"
 Meshes = "0.53, 0.54, 0.55, 0.56"
 Optim = "1.12"
 PolynomialBases = "0.4.25"
-PreallocationTools = "0.4.26"
+PreallocationTools = "0.4.26, 1"
```

```suggestion
PreallocationTools = "1"
```
Still using PreallocationTools.jl <1. Let's see why.